packages/core/src/model.ts

```diff
@@ -163,6 +163,14 @@ export class ChartModel {
 			displayData,
 			(datum) => datum[domainIdentifier]
 		).keys();
+
+		// Sort keys
+		stackKeys.sort((a, b) => {
+			const dateA: any = new Date(a);
+			const dateB: any = new Date(b);
```

Comment:
are a & b always going to be strings of date values? will they ever be actual Date objects? do we need a check here?

Reply:
Since `const stackKeys = map().keys()` return an string array, we don't have to check the type of a & b again.